Single_Client/client.py

```diff
@@ -1,7 +1,9 @@
 import os
 import socket
 import subprocess
-
+import getpass
+import platform
+import os
```

Comment:
I was actually working on that similar goal a few weeks ago but hit a wall wld u like to see